assets/helpers/user/user.js

```diff
@@ -51,7 +51,7 @@ const init = (dispatch: Function) => {
     dispatch(setPostDeploymentTestUser(true));
   }
 
-  if (getCookie('gu_recurring_contributor')) {
+  if (getCookie('gu_recurring_contributor') === 'true') {
```

Comment:
this model feels a bit error-prone. could we wrap this cookie check in its own function? `isRecurringContributor`? maybe with a corresponding setter function?